### PR TITLE
chore: Improve measuring webpack performance

### PR DESCRIPTION
**Pull request by dmtrKovalenko**
Just more useful functionality

### CODE REVIEW

File: npm/webpack-dev-server/src/measureWebpackPerformance.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export function measureWebpackPerformance (webpackConfig: webpack.Configuration)
   const compareWithPrevious = process.env.WEBPACK_PERF_MEASURE_COMPARE
 
   function percentageDiff (a: number, b: number) {
```
Comment (elevatebart) [inline npm/webpack-dev-server/src/measureWebpackPerformance.ts:15]: Maybe using the word `before` and `after` would make this function easier to use.

**Review by elevatebart — APPROVED** [review]
LGTM

File: npm/webpack-dev-server/src/measureWebpackPerformance.ts (hunk 1 above)
Comment (dmtrKovalenko) [reply npm/webpack-dev-server/src/measureWebpackPerformance.ts:15]: It's not actually before and after but any number. Maybe base and comparing?

**Review by lmiller1990 — APPROVED** [review]
Great! You findings will make a great blog post or conf talk.

**Pull request merged**